### DISCUSSION TITLE
Amazon Linux 2023

### DISCUSSION THREAD

**Discussion by mic345** [chef/bento#1557] [Ideas]
Hi there and good morning 😊☀️

Are there any plans to add support for Amazon Linux 2023? They are planning to sunset Amazon Linux 2.

Take care and have a wonderful day!
Mickey.

**Comment by Stromweld** [comment]
That is the goal, just going to take some time to make that work.

**Comment by Stromweld** [comment]
sorry forgot about this discussion. AMZ linux 2023 exists in hcp cloud vagrant registry. bento/amazonlinux-2023 for virtualbox x86_64. https://portal.cloud.hashicorp.com/vagrant/discover/bento/amazonlinux-2023
